WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+
+final class WooShippingCustomsFormViewModel: ObservableObject {
+    @Published var internationalTransactionNumber: String
+    @Published var returnToSenderIfNotDelivered: Bool
+
+    let contentType: WooShippingContentType = .merchandise
+    let restrictionType: WooShippingRestrictionType = .none
+
+    let itnInfoURL = URL(string: "https://pe.usps.com/text/imm/immc5_010.htm")
```

Comment:
I see that we have this url in 2 more places, could we maybe put it to some URL extension or some other place where we would have it defined once and referenced in places where we need it?